### PR TITLE
responsiveness and container fix csoc

### LINKED ISSUE #31: in CSOC/Resources section "Join the force" tab is not correctly aligned
Screenshot (107)

### PR DESCRIPTION
**Pull request by king-11**
fix #31 and responsive footer

### CODE REVIEW

File: pages/csoc/resources.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
-  <div>
+  <v-container fluid>
```
Comment (dasari810) [inline pages/csoc/resources.vue:2]: @king-11  i don't think that resolved the issue completely , still some part of content is  over lapping

**Review by dasari810 — CHANGES_REQUESTED** [review]
i think the issue is not only in CSoC page but all pages have same issue , so i think it should be updated in default layout

**Comment by king-11** [comment]
🤔 yeah but like we need a wrapper for each page even if we include it in layout we will need another div for each page while we can ensure to use container with fluid on further pages.

File: pages/csoc/resources.vue (hunk 1 above)
Comment (king-11) [reply pages/csoc/resources.vue:2]: I tried reloading the page several times didn't happen can you post an ss

Comment (dasari810) [reply pages/csoc/resources.vue:2]: Just try navigating to blog then to csoc page .. then you can notice the issue

Comment (king-11) [reply pages/csoc/resources.vue:2]: The issue persists I guess with blog is the issue that it's 404 page whose contents aren't wrapped in v-main or container

**Review by dasari810 — APPROVED** [review]
LGTM  👍

**Pull request merged**